protocol/x/clob/types/keys.go

```diff
@@ -101,10 +105,6 @@ const (
 	// are stored in a transient store. This count represents the number of uncommitted stateful
 	// `placements - cancellations`.
 	UncommittedStatefulOrderCountPrefix = "NumUncmtLT:"
```

Comment:
This is all up to you, fine with leaving as-is if you prefer.

Should we update `UncommittedStatefulOrderCountPrefix` to be SO instead of LT?

Same with `UncommittedStatefulOrderCancellationKeyPrefix` and `UncommittedStatefulOrderPlacementKeyPrefix`.